src/app/reports/reports.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnDestroy } from "@angular/core";
+import { NavigationEnd, Router } from "@angular/router";
+import { Subscription } from "rxjs";
+import { filter } from "rxjs/operators";
+
+@Component({
+  selector: "app-reports",
+  templateUrl: "reports.component.html",
+})
+export class ReportsComponent implements OnDestroy {
+  homepage = true;
+  subscription: Subscription;
+
+  constructor(router: Router) {
+    this.subscription = router.events
```

Comment:
I was just researching this - seems pretty wild that this is the most definitive way to ask "where am I?" instead of what seems like more obvious ways like checking the `ActivatedRoute` or even the `Router` - neither of which have consistent ways of accessing that specific data :)

Reply:
We have to continue to listen for events since this component hangs around for each report.